lms/models/__init__.py

```diff
@@ -14,8 +14,9 @@
     'LtiLaunches',
     'build_from_lms_url',
     'build_from_lti_params',
-    'find_by_state',
+    'find_lti_params',
     'find_or_create_from_user',
+    'find_user_from_state',
```

Comment:
`find_user_from_state()` was part of the `lms.models` public interface but not properly exported, so I've added it here. (Other parts of the code that _import_ it still do so incorrectly though -- generally speaking there are lots of exports and imports, and lack of defined public API, that need fixing throughout the app.)